_posts/2016/2016-03-17-team-orange-mallorca.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "Developer team enjoys an innovative and productive offsite week on Mallorca"
+date: "2016-03-17 16:03:17"
+icon: suitcase
+categories: working-at-epages, events
+authors: ["Alejandra, Benjamin and Heiko"]
+---
+
+Last week, the ePages release and test automation team was welcomed by the precious views, hearty people and refreshing atmosphere of Mallorca. The purpose of this particular trip was to celebrate an offsite team event liberated from the daily tasks and routines of the business life. The team set off to new shores to develop a master plan on how it could face the winds of change of the advancement of each team member. The ultimate goal was to find the best way for contributing to the promising future of the whole research and development department.
+
+Now we are back and ready to briefly tell you the tale of the fearless developers' journey to a mysterious island. Enjoy the multifaceted report!
+
+## Travelling together
+
+With much curiosity we first took some time off and enjoyed the amazing view over the beach skyline from the terrace of our finca, [Son Pages](http://sonpages.com). On day two, we visited the [Cuevas del Drach](http://www.cuevasdeldrach.com) and the beautiful yacht harbour at [Porto Cristo](http://www.portocristo.org/) to draw fresh inspiration. On the next day, we undertook a road trip with the team van to Sa Calobra in the north. We were delighted by the majestic mountains and completely captivated by the bluish tonalities of the Mediterranean Sea. Alongside the curvy mountain road and nested into the picturesque bay were various [Geocaching](https://www.geocaching.com) possibilites that we thankworthy explored with tremendous pleasure.
```

Comment:
This whole paragraph reads very romantic. Can we skip some shiny adjectives here and there?
Suggestion: We were impressed by the majestic mountains and the bluish tonalities of the Mediterranean Sea.
When cruising along the bay area, we also found various [Geocaching](https://www.geocaching.com) possibilities to explore.